Bearded.Utilities/IO/Logger.cs

```diff
@@ -272,17 +272,17 @@ private static void ensureListCapacity(List<Entry> list, int neededCapacity)
 
         public Logger(ICollection<Severity> enabledSeverities)
         {
-            Fatal = getWriterIfSeverityEnabled(enabledSeverities, Severity.Fatal);
-            Error = getWriterIfSeverityEnabled(enabledSeverities, Severity.Error);
-            Warning = getWriterIfSeverityEnabled(enabledSeverities, Severity.Warning);
-            Info = getWriterIfSeverityEnabled(enabledSeverities, Severity.Info);
-            Debug = getWriterIfSeverityEnabled(enabledSeverities, Severity.Debug);
-            Trace = getWriterIfSeverityEnabled(enabledSeverities, Severity.Trace);
+            Fatal = createWriterIfSeverityEnabled(enabledSeverities, Severity.Fatal);
+            Error = createWriterIfSeverityEnabled(enabledSeverities, Severity.Error);
+            Warning = createWriterIfSeverityEnabled(enabledSeverities, Severity.Warning);
+            Info = createWriterIfSeverityEnabled(enabledSeverities, Severity.Info);
+            Debug = createWriterIfSeverityEnabled(enabledSeverities, Severity.Debug);
+            Trace = createWriterIfSeverityEnabled(enabledSeverities, Severity.Trace);
 
             RecentEntries = lines.AsReadOnly();
         }
 
-        private Writer getWriterIfSeverityEnabled(ICollection<Severity> enabledSeverities, Severity severity)
+        private Writer? createWriterIfSeverityEnabled(ICollection<Severity> enabledSeverities, Severity severity)
```

Comment:
The ? is obvious, but I also renamed it because it creates, it does not 'get' a writer.